### DISCUSSION TITLE
Shortcut Maker Compatibility

### DISCUSSION THREAD

**Discussion by Hamster45105** [MM2-0/Kvaesitso#454] [Q&A] [answered]
Hello! I was just wondering how I add shortcuts I have made in the "Shortcut Maker" (https://play.google.com/store/apps/details?id=rk.android.app.shortcutmaker) app to my favourites in the launcher.

Thanks

**Comment by MM2-0** [comment]
I'm not familiar with that app, how would you do it in other launchers?

**Reply by Hamster45105** [reply]
You would make a shortcut, click "Create Shortcut" and an icon would be added to your homescreen linking to whatever you made your shortcut go to.

**Reply by MM2-0** [reply, marked as answer]
That should work. The created shortcut should appear in the favorites section. Another way to create shortcuts in the launcher is to tap the edit button in the favorites panel, and then tap the plus icon in either of the pinned sections.
